docs/protocol/js-ceramic/guides/ceramic-nodes/running-locally.md

```diff
@@ -31,22 +56,41 @@ Open your console and install the CLI using npm:
 npm install -g @ceramicnetwork/cli
 ```
 
-### Launch the Ceramic node
+### Launch the `js-ceramic` node
 
-Use the `ceramic daemon` command to start a local JS Ceramic node connected to the [Clay Testnet](../../networking/networks.md#clay-testnet) at `https://localhost:7007`.
+Use the `ceramic daemon` command to start a local JS Ceramic node connected to the [Clay Testnet](../../networking/networks.md#clay-testnet) by default running at `https://localhost:7007`:
 
 ```bash
 ceramic daemon
 ```
 
-### **Configure your network**
+### Configure your network
 
 (Optional) By default, the JS CLI starts a node on the [Clay Testnet](../../networking/networks.md#clay-testnet). If you would like to use a different network, you can specify this using the `--network` option. View [available networks](../../networking/networks.md). Note, the CLI can not be used with [Mainnet](../../networking/networks.md#mainnet).
 
-### **Configure a node URL**
+### Configure a node URL
 
 (Optional) It is possible to use the CLI with a remote Ceramic node over HTTP, instead of a local node. To do this, use the `config set` command to set the `ceramicHost` variable to the URL of the node you wish to use.
 
 ```bash
 ceramic config set ceramicHost 'https://yourceramicnode.com'
 ```
+
+## Monitoring
+You can always check if `js-ceramic` and `rust-ceramic` components are available by running the commands listed below.
```

Comment:
We should probably rename `rust-ceramic` to `ceramic-one` in the simpledeploy repo.